modules/era/sql/disable_walking_into_cop_zones.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE zonelines SET tozone = 196, tox = 100.007, toy = -61.630, toz = -237.441, rotation = 187 WHERE zoneline = 845624698;     -- Gusgen -> Oldonton
+UPDATE zonelines SET tozone = 118, tox = -276.529, toy = 16.403, toz = -324.519, rotation = 14 WHERE zoneline = 945894266;      -- Buburimu -> Bibiki
+UPDATE zonelines SET tozone = 198, tox = -140.246, toy = -12.738, toz = 160.709, rotation = 63 WHERE zoneline = 879310202;      -- Shakhrami -> Attohwa
+UPDATE zonelines SET tozone = 112, tox = -136.287, toy = -23.268, toz = 137.302, rotation = 91 WHERE zoneline = 875836282;      -- Xarcabard -> Uleguerand
+UPDATE zonelines SET tozone = 104, tox = 342.000, toy = -5.000, toz = 15.117, rotation = 169 WHERE zoneline = 947335802;        -- Jugner -> Carpenters (J-8)
+UPDATE zonelines SET tozone = 104, tox = -536.152, toy = 1.163, toz = 336.587, rotation = 43 WHERE zoneline = 1635201658;       -- Jugner -> Carpenters (E-6)
```

Comment:
Northern Sandoria has a path to Carpenter's Landing as well: 

![image](https://user-images.githubusercontent.com/14808841/196216001-2f6ab351-48a5-4cba-950f-c24f3f435864.png)